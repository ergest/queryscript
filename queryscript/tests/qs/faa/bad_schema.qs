export type AircraftModel {
    number_value bigint,
}

-- This should throw a runtime error
export let aircraft_models [AircraftModel] = load('data/aircraft_models.parquet');

SELECT MAX(number_value) FROM aircraft_models;
