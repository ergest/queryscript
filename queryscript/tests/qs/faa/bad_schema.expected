{
    "compile_errors": [],
    "decls": {
        "let aircraft_models": [{
        	number_value Int64,
        }],
        "type AircraftModel": {
        	number_value Int64,
        },
    },
    "queries": [
        Err(
            StringError {
                what: "Parquet file \"tests/qs/faa/data/aircraft_models.parquet\" has a different schema than the target variable's type",
                backtrace: None,
            },
        ),
    ],
}